src/domain/synthetics/markets/useMarketsInfoRequest.ts

```diff
@@ -93,307 +103,591 @@ export function useMarketsInfoRequest(chainId: number): MarketsInfoResult {
-              },
-              swapFeeFactorForNegativeImpact: {
-                methodName: "getUint",
-                params: [swapFeeFactorKey(marketAddress, false)],
-              },
-              swapImpactFactorPositive: {
-                methodName: "getUint",
-                params: [swapImpactFactorKey(marketAddress, true)],
-              },
-              swapImpactFactorNegative: {
-                methodName: "getUint",
-                params: [swapImpactFactorKey(marketAddress, false)],
-              },
-              swapImpactExponentFactor: {
-                methodName: "getUint",
-                params: [swapImpactExponentFactorKey(marketAddress)],
-              },
-              longInterestUsingLongToken: {
-                methodName: "getUint",
-                params: [openInterestKey(marketAddress, market.longTokenAddress, true)],
-              },
-              longInterestUsingShortToken: {
-                methodName: "getUint",
-                params: [openInterestKey(marketAddress, market.shortTokenAddress, true)],
-              },
-              shortInterestUsingLongToken: {
-                methodName: "getUint",
-                params: [openInterestKey(marketAddress, market.longTokenAddress, false)],
-              },
-              shortInterestUsingShortToken: {
-                methodName: "getUint",
-                params: [openInterestKey(marketAddress, market.shortTokenAddress, false)],
-              },
-              longInterestInTokensUsingLongToken: {
-                methodName: "getUint",
-                params: [openInterestInTokensKey(marketAddress, market.longTokenAddress, true)],
-              },
-              longInterestInTokensUsingShortToken: {
-                methodName: "getUint",
-                params: [openInterestInTokensKey(marketAddress, market.shortTokenAddress, true)],
-              },
-              shortInterestInTokensUsingLongToken: {
-                methodName: "getUint",
-                params: [openInterestInTokensKey(marketAddress, market.longTokenAddress, false)],
-              },
-              shortInterestInTokensUsingShortToken: {
-                methodName: "getUint",
-                params: [openInterestInTokensKey(marketAddress, market.shortTokenAddress, false)],
-              },
-              virtualMarketId: {
-                methodName: "getBytes32",
-                params: [virtualMarketIdKey(marketAddress)],
-              },
-              virtualLongTokenId: {
-                methodName: "getBytes32",
-                params: [virtualTokenIdKey(market.longTokenAddress)],
-              },
-              virtualShortTokenId: {
-                methodName: "getBytes32",
-                params: [virtualTokenIdKey(market.shortTokenAddress)],
-              },
+        };
+
+        performance.mark("useMarketsInfo-request-build-market-hash-start");
+        const hashedKeys = await hashDataMapAsync(
```

Comment:
so what happens here is that we get hashed keys from one worker and send it to another worker.

I think we should extend worker multicalling function the way that it accepts unhashed keys, like you doing here. That way hash worker is not needed, and we can leave just a one worker for multicalls